tutorials/docs-09-using-turing-advanced/advanced.jmd

```diff
@@ -176,6 +176,59 @@ gdemo(x) = Turing.Model(gdemo, (; x))
 model = gdemo([1.5, 2.0])
 ```
 
+### Reparametrization and generated_quantities
+
+Often, the most natural parameterization for a model is not the most computationally feasible. Consider the following
+(efficiently reparametrized) implementation of Neal's funnel [(Neal, 2003)](https://arxiv.org/abs/physics/0009028):
+
+```julia; eval=false
```

Comment:
Would it be worth making `eval=true` here? Would potentially be nice to have the outputs of this example if it doesn't take too long to run:)